boards/arm/npcx7m6fb_evb/Kconfig.defconfig

```diff
@@ -10,3 +10,6 @@ endif # BOARD_NPCX7M6FB_EVB
 
 config SYS_CLOCK_TICKS_PER_SEC
 	default 1000
+
+config INPUT
+	default y if KSCAN
```

Comment:
Somewhat related to this change, but can you update the Input subsystem documentation with a template for how to update an existing Kscan implementation to the Input subsystem?

Looks like it can be summarized as:
* CONFIG_INPUT=y
* Update chosen node assignment of `zephyr,keyboard-scan`
* Add a childnode with compat `zephyr,kscan-input` to the existing devicetree node.

This should be done as a separate PR.  Also consider adding a diagram that shows how the input events, input devices, and application API work together.

Reply:
Yeah good idea, opened https://github.com/zephyrproject-rtos/zephyr/pull/57956